### PR TITLE
Add C-CDA to FHIR doc

### PR DESCRIPTION
**Pull request by isuruuy429**
This PR contains the scenario of CCDA to FHIR data transformation.

### CODE REVIEW

File: en/docs/data-transformation/guides/ccda-to-fhir.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+## Introduction
+The health.fhir.r4utils.ccdatofhir library is a Ballerina module designed to facilitate the conversion of Clinical Document Architecture (CDA) documents, specifically the Continuity of Care Document (CCD-A) format, into FHIR R4 resources. In the healthcare industry, data exchange between systems using different standards is a significant challenge. The CCD-A format, widely used for clinical documents, often needs to be integrated into systems that utilize the more modern FHIR R4 standard for interoperability and data exchange.
```
Comment (sameeragunarathne) [inline en/docs/data-transformation/guides/ccda-to-fhir.md]: Let's not specify about ballerina here, in the ballerina tab we should have specifics

File: en/docs/data-transformation/guides/ccda-to-fhir.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+This library provides a streamlined and efficient way for developers to convert CCD-A documents into the FHIR R4 format, enabling seamless integration with FHIR-based healthcare systems. 
+
+## Mapping C-CDA Patient to FHIR r4
+The body of a C-CDA document contains the clinical content, which is organized into structured sections. Each section corresponds to different types of clinical information, such as Patient information, Immunizations, Medications, Allergies, Problems, Procedures. Each section in the body is represented as an XML element with a specific structure, typically including a title, code (for standardized representation), and entries. Entries are sub-elements that provide detailed information, such as specific medications or results.
+
```
Comment (sameeragunarathne) [inline en/docs/data-transformation/guides/ccda-to-fhir.md:8]: Need to have the note of implementation flavours and the tab for ballerina

{!includes/bal-mi-note.md!}

File: en/docs/data-transformation/guides/ccda-to-fhir.md (hunk 1 above)
Comment (isuruuy429) [reply en/docs/data-transformation/guides/ccda-to-fhir.md]: Fixed.

File: en/docs/data-transformation/guides/ccda-to-fhir.md (hunk 2 above)
Comment (isuruuy429) [reply en/docs/data-transformation/guides/ccda-to-fhir.md:8]: Fixed with https://github.com/wso2/open-healthcare-docs/pull/30/commits/27b3f1e24f0af2adcea6281dd0c9ca7d5c471b43

**Review by sameeragunarathne — DISMISSED** [review]

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**